modules/infrastructure-moving-monitoring.adoc

```diff
@@ -24,39 +28,87 @@ metadata:
 data:
   config.yaml: |+
     alertmanagerMain:
-      nodeSelector:
+      nodeSelector: <1>
         node-role.kubernetes.io/infra: ""
+      tolerations:
+      - key: node-role.kubernetes.io/infra
+        value: reserved
+        effect: NoSchedule
+      - key: node-role.kubernetes.io/infra
+        value: reserved
+        effect: NoExecute
     prometheusK8s:
       nodeSelector:
         node-role.kubernetes.io/infra: ""
+      tolerations:
+      - key: node-role.kubernetes.io/infra
+        value: reserved
+        effect: NoSchedule
+      - key: node-role.kubernetes.io/infra
+        value: reserved
+        effect: NoExecute
     prometheusOperator:
       nodeSelector:
         node-role.kubernetes.io/infra: ""
+      tolerations:
+      - key: node-role.kubernetes.io/infra
+        value: reserved
+        effect: NoSchedule
+      - key: node-role.kubernetes.io/infra
+        value: reserved
+        effect: NoExecute
     k8sPrometheusAdapter:
```

Comment:
missed `grafana` component for 4.6 - 4.10, see from:
https://docs.openshift.com/container-platform/4.10/machine_management/creating-infrastructure-machinesets.html#infrastructure-moving-monitoring_creating-infrastructure-machinesets

from 4.11, grafana is removed from stack, no configuration for it is fine
https://docs.openshift.com/container-platform/4.11/machine_management/creating-infrastructure-machinesets.html#infrastructure-moving-monitoring_creating-infrastructure-machinesets

Reply:
I will manually cherrypick these changes and add the `grafana` component there.